config/routes.rb

```diff
@@ -17,7 +17,9 @@
     get '/dashboard/edit', to: 'users#edit', as: :dashboard_edit
     patch '/dashboard/edit', to: 'users#update', as: :dashboard_patch
 
-    get '/dashboard/change_password', to: 'passwords#edit', as: :password_edit
+    get '/dashboard/change_password', to: 'passwords#reset', as: :password_reset
```

Comment:
to make this more RESTful, i don't know if i'd nest it under "dashboard" - is it related to dashboard at all? i'd maybe suggest "/passwords/new" since we're "reseting" it